contracts/smart-account/modules/MultiOwnedECDSAModule.sol

```diff
@@ -55,9 +55,7 @@ contract MultiOwnedECDSAModule is
                 );
 
             _smartAccountOwners[eoaOwners[i]][msg.sender] = true;
-            unchecked {
```

Comment:
having this way wouldn't it be more optimised?

Reply:
it's handled by Solidity since 0.8.22